src/data/battler-tags/grounded-tag.ts

```diff
@@ -12,7 +12,8 @@ import i18next from "i18next";
 
 /**
  * Tag for effects that ground the source, allowing Ground-type moves to hit them.
- * @description `IGNORE_FLYING`: Persistent grounding effects (i.e. from Smack Down and Thousand Waves)
+ *
+ * `IGNORE_FLYING`: Persistent grounding effects (i.e. from Smack Down and Thousand Waves)
```

Comment:
why is this called grounded but the tag type is `IGNORE_FLYING`. That comment was extremely confusing
```suggestion
 * Corresponds to `BattlerTagType.IGNORE_FLYING`
```
or a `@linkcode`, or `@see`?

Reply:
How about this?
```ts
/**
 * Tag for effects that ground the source (i.e. from Smack Down and Thousand Waves),
 * allowing Ground-type moves to hit them.
 *
 * @see {@linkcode BattlerTagType.IGNORE_FLYING}
 * @extends BattlerTag
 */
```

Reply:
looks good